persistence/src/main/resources/reference.conf

```diff
@@ -79,6 +79,8 @@ lagom.persistence.cluster.distribution {
   ensure-active-interval = 30s
 }
 
+akka.logger-startup-timeout = 25s
```

Comment:
You should never "override" akka configuration from `reference.conf` since the result depends on classpath ordering. Instead you should do that in play overrides.

I guess this is because of the osx issue with localhost. It's better to let the users know and fix their environment instead of always having a slow startup. See https://github.com/akka/akka/issues/22160 where the conclusion is that we should improve the log message in Akka.